contracts/src/systems/cc_map/contract.cairo

```diff
@@ -0,0 +1,141 @@
+// ---------------------------------------- contract --------------------------------------
```

Comment:
this shouldn't exist in this repo either

Reply:
This is dojo::contract. We are not able to import the package by dependency, we need to have an implementation in the repo.